.gitignore

```
@@ -1,5 +1,5 @@
 .DS_Store
-/.build
+**/.build/**
```

```suggestion
.build
```
I believe that's equivalent.